apps/web/src/app/core/components/header/header.component.html

```diff
@@ -22,25 +22,6 @@
           />
         </svg>
       </button>
-      <button class="btn btn-ghost btn-circle">
-        <div class="indicator">
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            class="h-5 w-5"
-            fill="none"
-            viewBox="0 0 24 24"
-            stroke="currentColor"
-          >
-            <path
```

Comment:
Aqui acho que dava pra adotar o InlineSVG e fazer a substituição desse path
https://www.npmjs.com/package/ng-inline-svg

Reply:
Pior que esse pacote aí tá como _deprecated_.

Tem uma parada que eu fiz no trabalho que é criar um componente para apresentar SVGs permitindo alteração de cor e mantendo o esquema de caching que utilizar o inline SVG proporciona (por mais que seja meio feinho, ele dá uma pequena vantagem de performance já que o browser não vai fazer uma request para buscar o arquivo .svg nos assets).

Aqui nesse tópico ele explica um pouco melhor de como funciona. Talvez vale aplicar, o que acha?
### [Best way to use SVG icons in Angular with caching and possibility of changing colour](https://medium.com/@rado.sabo/best-way-to-use-svg-icons-in-angular-with-caching-and-possibility-of-changing-colour-71923bb3f189)

Vale lembrar que a biblioteca [DaisyUI](https://daisyui.com/) (o framework CSS que estamos aplicando aqui) trabalha com um design system que consome o [_**ionicons5**_](https://www.xicons.org/#/).